### DISCUSSION TITLE
GSoC’24 Proposal Review | 4.2 Multiple classrooms support (Vishwajith Shettigar).

### DISCUSSION THREAD

**Discussion by Vishwajith-Shettigar** [oppia/oppia-android#5360] [Proposal reviews (4. Android projects)]
Hi,

Please review my proposal for Multiple classrooms support project's **Section 1: WHAT** .
Here is my [doc link](https://docs.google.com/document/d/19-Gv0k7gNe6TK5H8vYpl5M2jgEI3LtdqiVZynzxev9M/edit?usp=sharing). gsoc-2024-mentors@oppia.org added as editor.

Waiting for the feedback.
Thank you.

**Comment by Vishwajith-Shettigar** [comment]
Hi @adhiamboperes,
Please review **How section** up to risks and mitigations .
Here is [doc link](https://docs.google.com/document/d/19-Gv0k7gNe6TK5H8vYpl5M2jgEI3LtdqiVZynzxev9M/edit?usp=sharing). [gsoc-2024-mentors@oppia.org](mailto:gsoc-2024-mentors@oppia.org) added as an editor.

Waiting for the feedback.
Thank you.

**Comment by BenHenning** [comment]
Hi. Apologies for the late follow-up. Per the project tracking issue (https://github.com/oppia/oppia-android/issues/5345#issuecomment-1988936046) project 4.3 is no longer a recommended GSoC project.

You're welcome to continue with a proposal, but we can't guarantee that there will be a mentor available for this project. I suggest that you instead consider looking into one of the other projects (code coverage or multiple classrooms) as these have committed mentors available.

**Reply by Vishwajith-Shettigar** [reply]
@BenHenning, you commented on wrong proposal review I guess.

**Reply by BenHenning** [reply]
I misread which project this corresponded to--my apologies!

**Comment by Vishwajith-Shettigar** [comment]
Hi, I have completed the rest of my proposal. Please have a look. Waiting for suggestions.
[doc link](https://docs.google.com/document/d/19-Gv0k7gNe6TK5H8vYpl5M2jgEI3LtdqiVZynzxev9M/edit?usp=sharing)
Thank you
